test/WebJobs.Script.Tests.E2E/FunctionAppFixture.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using Xunit;
+
+namespace WebJobs.Script.EndToEndTests
+{
+    public class FunctionAppFixture : IDisposable
```

Comment:
As a collection fixture, this would be reused across all tests (or all tests using the same collection), I do generate a session ID that is specific to a test run in the `TelemetryContext` here: https://github.com/fabiocav/azure-webjobs-sdk-script/blob/tst/test/WebJobs.Script.Tests.E2E/TelemetryContext.cs#L17